frontend/src/metabase/visualizations/click-actions/types.ts

```diff
@@ -1,11 +1,15 @@
 import type React from "react";
```

Comment:
What do you think about moving this to the `visualizations/types/click-actions`? Larger visualization types depend on it, I cleaned it up here https://github.com/metabase/metabase/pull/33196/commits/a72f508d9adbca336507d29bfc4450dc1ba22815#diff-e12972f3eb6bc1414f6c907c599e8b35523cf003157538982abbf544a075b54c

I can do it myself after you merge the PR